test/tools/libtesteth/ImportTest.cpp

```diff
@@ -524,13 +524,21 @@ void ImportTest::parseJsonStrValueIntoVector(json_spirit::mValue const& _json, v
 
 void parseJsonIntValueIntoVector(json_spirit::mValue const& _json, vector<int>& _out)
 {
+	auto converter = [&_out](json_spirit::mValue const& _value)
+	{
+		if (_value.type() == json_spirit::int_type)
+			_out.push_back(_value.get_int());
+		if (_value.type() == json_spirit::str_type)
+			_out.push_back(std::atoi(_value.get_str().c_str()));
+	};
+
 	if (_json.type() == json_spirit::array_type)
 	{
 		for (auto const& val: _json.get_array())
-			_out.push_back(val.get_int());
+			converter(val);
 	}
 	else
-		_out.push_back(_json.get_int());
+		converter(_json);
```

Comment:
I don't think it's a good idea to allow both int and str here. Let's just pick one of these and stick to that.  Maybe an `int`.

Reply:
well. yaml does not know which type of the field it's parsing. and assume that it is string. which is true for all our json fields. but this one. we already have all tests with this fields as int. 
but after yaml converts this into json it becomes string. so either I force this field back to string by adding a new function or we keep this change which allows integer array in a string form.

Reply:
You can try this https://github.com/ethereum/tests/issues/369#issuecomment-352452404